### PR TITLE
remove unused deps in package.json

### LINKED ISSUE #669: audit package.json and remove unused deps
just realized we still have some unused deps in our `package.json`.

Two that I noticed were `coverify` should be in dev-deps and `request` should be removed all together.

### PR DESCRIPTION
**Pull request by meandavejustice**
- fixes #669
- remove `request`
- remove `jsonwebtoken`
- move `coverify` from dependencies to devDependencies
- upgrade `ampersand-events` 1.1.1 > 2.0.1
- upgrade `ampersand-model` 6.0.2 > 7.0.0
- upgrade `ampersand-state` 5.0.0 > 5.0.1
- upgrade `ampersand-view` 9.0.2 > 10.0.0
- upgrade `js-cookie` 2.0.3 > 2.1.1
- upgrade `query-string` 3.0.3 > 4.1.0

### CODE REVIEW

File: testpilot/frontend/static-src/app/views/landing-page.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default PageView.extend({
   render() {
     const isLoggedIn = !!app.me.user.id;
     const query = queryString.parse(location.search);
-    this.isMoz = query.hasOwnProperty('butimspecial') || query.hasOwnProperty('butimspecial/');
+    this.isMoz = ('butimspecial' in query);
```
Comment (meandavejustice) [inline testpilot/frontend/static-src/app/views/landing-page.js:24]: https://github.com/sindresorhus/query-string/commit/7bebd8e430eb41ed42a749a1dd1d546eeb7fedb5

**Comment by lmorchard** [comment]
Might be worth splitting this into two PRs - i.e. one for removing unused deps, and another for upgrading used deps. Upgrading ampersand stuff feels riskier than removing unused stuff, but I might just be superstitious.

(That said, I'm still going to poke at things and probably just merge if I don't see any issues)

**Comment by lmorchard** [comment]
Eh, no errors or problems I can see, let's merge

**Comment by meandavejustice** [comment]
thanks @lmorchard

**Pull request merged**